client/my-sites/customer-home/cards/features/go-mobile/style.scss

```diff
@@ -6,9 +14,16 @@
 	width: 50%;
 }
 
+.go-mobile__email-link {
+	margin: 24px 0 0;
+	padding-top: 20px;
+	border-top: 1px solid var( --color-neutral-5 );
+	color: var( --color-text-subtle );
+}
+
 .go-mobile__email-link-button {
-	margin-top: 1rem;
-	width: 100%;
+	cursor: pointer;
```

Comment:
The pointer cursor is already defined in `.button` so I don't think we need it here:

<img width="255" alt="Screen Shot 2020-05-08 at 10 45 27" src="https://user-images.githubusercontent.com/1233880/81388774-1160fc00-9119-11ea-95cb-7b2f7d2b2753.png">